include/graph/graph.hpp

```diff
@@ -59,17 +28,17 @@ class Graph {
     arrayV_.push_back(0);
     V_ = 0;
   }
-  void setInput(LayerExample& lay, const std::vector<int>& vec) {
+  void setInput(Layer& lay, Tensor& vec) {
```

Comment:
if you will change `Tensor` to `std::vector<Tensor>` then please change this same